site/DataPageV2Content.tsx

```diff
@@ -822,7 +822,11 @@ const KeyDataTable = (props: {
                     <div className="key-data__title key-data-description-short__title">
                         {datapageData.title}
                     </div>
-                    <div>{datapageData.descriptionShort}</div>
+                    <div>
+                        <SimpleMarkdownText
```

Comment:
The new renderer by default wraps any text into a paragraph which adds unwanted vertical space. I added an option called `useParagraphs` for single-liners specifically that – if set to false – disables that beaviour. It would be good to use that here. 

```jsx
<SimpleMarkdownText text={...} useParagraphs={false} />
```

Reply:
Or, alternatively, if we expect `descriptionShort` be have multiple paragraphs, then it would be good to set the `p` margins to 0 here.

Reply:
Good to know! The ETL guidelines say that this should be a single paragraph so I'll go with useParagraphs=false. Thank you!